src/java.desktop/share/classes/java/awt/Component.java

```diff
@@ -8974,8 +8949,6 @@ private void readObject(ObjectInputStream s)
     {
```

Comment:
`@SuppressWarnings("removal")` can be removed

Reply:
ok